### PR TITLE
Feat/utilisation chart on dashboard

### LINKED ISSUE #255: Add utilization chart
This should plot number of children immunized, no of doses per antigen supplied per round.

I.e the X-axis should be by round, while the Y-axis should be line charts, each line representing number immunized and number of doses delivered in that round per antigen.

### PR DESCRIPTION
**Pull request by freefony**
closes  #255

### CODE REVIEW

**Comment by janetbutts** [comment]
closes #fix #255

File: src/app/components/delivery/daily-delivery.service.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ angular.module('delivery')
         params = pouchUtil.key(roundId)
       }
       params.include_docs = true
-
+      params.descending = true
```
Comment (tlvince) [inline src/app/components/delivery/daily-delivery.service.js]: The schedule page doesn't work with this set. PouchDB is throwing the following:

```
PouchError {status: 400, name: "query_parse_error", message: "Some query parameter is invalid", error: true, reason: "No rows can match your key range, reverse your start_key and end_key or set descending=false", …}
```

File: src/app/components/kpi/kpi.service.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ angular.module('kpi')
     _this.getByRoundId = function (id) {
       var view = 'kpi/by-round-id'
       var param = {
-        key: id,
+        keys: id,
```
Comment (tlvince) [inline src/app/components/kpi/kpi.service.js]: This should be `key` as we're only querying one ID.

**Comment by freefony** [comment]
@tlvince, fixed

**Comment by tlvince** [comment]
@freefony how do I test this? After creating a new round and marking some drops as successful, I'm just seeing the following:

screen shot 2015-11-11 at 13 24 10

**Comment by freefony** [comment]
did you enter kpi records for the round?

**Comment by tlvince** [comment]
Yeah got it :)

**Comment by freefony** [comment]
:+1:

**Comment by tlvince** [comment]
Yay numbers and figures:

screen shot 2015-11-11 at 13 37 11

**Pull request merged**